test/unit/src/common/components/select-repository-row/t_select-repository-row.js

```diff
@@ -0,0 +1,81 @@
+import expect from 'expect';
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import SelectRepositoryRow from '../../../../../../src/common/components/select-repository-row';
+
+describe('The SelectRepositoryRow component', () => {
+  let props;
+
+  beforeEach(() => {
+    props = {};
+  });
+
+  context('when a repository is supplied', () => {
+    beforeEach(() => {
+      props.repository = {
+        fullName: 'katiefenn/parker'
+      };
+    });
+
+    it('should display a label with the name of the repository', () => {
```

Comment:
It's heavily ingrained in me at the moment - we used them everywhere in my last job too. Agreed that we can fix this later.